libvirt/kcli_cluster.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Copyright Confidential Containers Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Manage Kubernetes clusters with kcli.
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CLUSTER_DISK_SIZE="${CLUSTER_DISK_SIZE:-20}"
+CLUSTER_MASTERS="${CLUSTER_MASTERS:-1}"
+CLUSTER_NAME="${CLUSTER_NAME:-peer-pods}"
+CLUSTER_IMAGE="${CLUSTER_IMAGE:-ubuntu2004}"
+CLUSTER_WORKERS="${CLUSTER_WORKERS:-1}"
+LIBVIRT_NETWORK="${LIBVIRT_NETWORK:-default}"
+LIBVIRT_POOL="${LIBVIRT_POOL:-default}"
+
+# Wait until the command return true.
+#
+# Parameters:
+#   $1 - wait time.
+#   $2 - sleep time.
+#   $3 - the command to run.
+#
+wait_for_process() {
+	local wait_time="$1"
+	local sleep_time="$2"
+	local cmd="$3"
+
+	while ! eval "$cmd" && [ "$wait_time" -gt 0 ]; do
+		sleep "$sleep_time"
+		wait_time=$((wait_time-"$sleep_time"))
+	done
+
+	[ "$wait_time" -gt 0 ]
```

Comment:
```suggestion
	[ "$wait_time" -ge 0 ]
```
I think this should be greater than of equal to, otherwise if the process passes in the last time slot then wait_time will be 0, so 0 -gt 0 will return 'false'

Reply:
good catch!